package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "ffpass",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
@d-Rickyy-b I think it was bumped in here as well.

Reply:
Seems like I'm blind :D